src/generic/GenericTypes.jl

```diff
@@ -475,15 +475,15 @@ const SparsePolyID = CacheDictType{Tuple{Ring, Symbol}, SparsePolyRing}()
 
 mutable struct SparsePoly{T <: RingElement} <: AbstractAlgebra.RingElem
    coeffs::Vector{T}
-   exps::Array{UInt}
+   exps::Vector{UInt}
```

Comment:
Herr in a struct. Maybe it was the only one, though? I forgot. Can't think well right now, fever is rising again

Reply:
Right, I missed that